manager/testdata/new-blueprints/transform-module.yaml

```diff
@@ -34,6 +34,7 @@ spec:
       level: column
   api:
     endpoint:
+      # Always equals the release name. It is mandatory if this module exposes a common service.
```

Comment:
What do you mean by release name here? If the module configures a common service this is a fixed endpoint for the service. If the module is a deployment itself this will not be specified and the endpoint will be the module release name instead. (As currently a name based on certain properties of the step)

Reply:
I meant it to be exactly what you described so I will change the comment to make it clearer Thanks.